apple-pie/styles/style.css

```
@@ -0,0 +1,33 @@
+
+
+.encabezado {
+    color: white;
+    font-size: 35px;
+    padding: 500px;
+    background-image: url(/apple-pie/images/apple-pie.jpg);
+    background-size: cover;
+    text-align: center
+}
+
+.texto {
+    margin: 50px 300px; 
```

```suggestion
    margin: 0 auto; 
    width: 70%;
```
Esta sentencia lo que hace es posicionar todo lo que le digas al centro, dandole un with (si por defecto es block tiene el 100% pero si se lo quieres especificar como hago lo puedes hacer